src/pages/api/modules/downloads.ts

```diff
@@ -53,7 +53,7 @@ async function Post(req: NextApiRequest, res: NextApiResponse) {
         ...(
           await new Transmission({
             baseUrl: service.url,
-            username: service.username,
+            username: 'username' in service ? service.username : '',
```

Comment:
I don't think this code has any use because `service.username` will just be undefined if not found and that works on the `New Client` statement, it just won't use a username. The way you are doing is using an empty username.I believe they should achieve the same result